spec-tests/src/test/java/org/candlepin/spec/owners/OwnerResourceSpecTest.java

```diff
@@ -315,8 +314,14 @@ void onlySuperAdminCanRefresh() throws ApiException {
         ApiClient readWriteClient = ApiClients.trustedUser(readWriteUser.getUsername());
         ApiClient adminClient = ApiClients.trustedUser(adminUser.getUsername());
 
-        ProductDTO product = createProduct(owner);
-        owners.createPool(owner.getKey(), Pools.random(product));
+        if (CandlepinMode.isHosted()) {
```

Comment:
Creation of a pool up or downstream is extraneous for this test as-written. We currently don't care if refresh does anything, just that we can invoke it.

Reply:
Done